plugins/scaffolder/src/components/fields/OwnerPicker/OwnerPicker.tsx

```diff
@@ -50,6 +51,8 @@ export const OwnerPicker = (
         'User',
       ]) as string[],
       defaultKind: 'Group',
+      allowArbitraryValues:
+        (uiSchema['ui:options']?.allowArbitraryValues as boolean) && true,
```

Comment:
```suggestion
        uiSchema['ui:options']?.allowArbitraryValues ?? true,
```

Reply:
Yep I revert to use the optional chaining parameter instead. 3rd time the charm.